tests/test_http.py

```diff
@@ -384,6 +384,10 @@ def test_cookie_domain_resolving(self):
         val = http.dump_cookie('foo', 'bar', domain=u'\N{SNOWMAN}.com')
         strict_eq(val, 'foo=bar; Domain=xn--n3h.com; Path=/')
 
+    def test_cookie_samesite(self):
+        val = http.dump_cookie('foo', 'bar', samesite=True)
```

Comment:
see other comments; `True` does not make much sense for this option so this test should cover both none, strict and lax mode.